server/modlog.ts

```diff
@@ -166,16 +166,14 @@ export class Modlog {
 		this.database.exec("PRAGMA foreign_keys = ON;");
 
 		// Set up tables, etc
-		this.database.exec(FS(MODLOG_SCHEMA_PATH).readIfExistsSync());
+		let tokenizer = 'unicode61';
 		if (Config.modlogftsextension) {
 			this.database.exec(`SELECT load_extension('native/fts_id_tokenizer.o')`);
-			this.database.exec(`CREATE VIRTUAL TABLE IF NOT EXISTS modlog_fts USING fts5(note, userid, autoconfirmed_userid, action_taker_userid, content=modlog, content_rowid=modlog_id, tokenize='id_tokenizer')`);
-			this.database.exec(`CREATE VIRTUAL TABLE IF NOT EXISTS alts_fts USING fts5(userid, content=alts, content_rowid=rowid, tokenize='id_tokenizer')`);
-		} else {
-			this.database.exec(`CREATE VIRTUAL TABLE IF NOT EXISTS modlog_fts USING fts5(note, userid, autoconfirmed_userid, action_taker_userid, content=modlog, content_rowid=modlog_id)`);
-			this.database.exec(`CREATE VIRTUAL TABLE IF NOT EXISTS alts_fts USING fts5(userid, content=alts, content_rowid=rowid)`);
+			tokenizer = 'id_tokenizer';
```

Comment:
This is backwards

Reply:
What do you mean?

Reply:
I intended the `Config.modlogftsextension` option to be set when we were using the built extension. 
```ts

let tokenizer = 'unicode61';
if (Config.modlogftsextension) {
	this.database.exec(`SELECT load_extension('native/fts_id_tokenizer.o')`);
	tokenizer = 'id_tokenizer'; 
 }
```
This is the code I currently have; I don't think it will use `id_tokenizer` unless it's built?

Reply:
Sorry, you are correct, I misread the unified diff